File: 14. Longest Common Prefix.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        if len(strs) == 0:
            return ""
        prefix = strs[0]
        for word in strs:
            while word.find(prefix) != 0:
                prefix = prefix[:-1]
                if prefix == '':
                    return ''
        return prefix
        

File: 14. Longest Common Prefix.php
class Solution {
    function longestCommonPrefix($strs) {
           if (count($strs) === 0) {
        return "";
    }
    $prefix = $strs[0];
    foreach ($strs as $word) {
        while (strpos($word, $prefix) !== 0) {
            $prefix = substr($prefix, 0, -1);
            if ($prefix === '') return '';
        }
    }
    return $prefix; 
    }
}
